apps/explorer/src/components/analytics/PostHogAnalyticsProvider.tsx

```diff
@@ -20,11 +20,10 @@ export function PostHogAnalyticsProvider({ children }: PostHogProviderProps) {
         <PostHogProvider
             apiKey="phc_IggVMJtR5vawlA4H3IIYnIyWjcK8rPiqAI1FlmKZPjp"
             options={{
-                // We'll set local storage as the default persistence method so
-                // that we don't have to show cookie banners in our applications
-                persistence: 'localStorage',
-                // We need to manually collect page view events since
-                // all of our applications use client-side routing
+                // We'll set memory as the default persistence method so that
+                // we aren't required to show a cookie acceptance banner
+                persistence: 'memory',
+                // We need to manually collect page view events since we use client-side routing
```

Comment:
updating these comments to be Explorer-specific as well since I forgot to change that in my last PR